client/app/components/DirectoryRow.js

```diff
@@ -2,15 +2,20 @@ import React from 'react';
 
 const DirectoryRow = (props) => {
   return (
-    <tr onClick={props.handleClick.bind(null, props.directory.name)}>
+    <tr onClick={props.handleClick.bind(null, props.name)}>
```

Comment:
one more thing you can do to *slightly clean up* might be doing 
`const { name, handleClick }  = props`
or doing 
`const DirectoryRow = ({ name, handleClick}) => {`